adam-core/src/test/scala/org/bdgenomics/adam/rdd/feature/CoverageRDDSuite.scala

```diff
@@ -58,14 +58,45 @@ class CoverageRDDSuite extends ADAMFunSuite {
 
     val coverage = sc.loadCoverage(outputFile)
     assert(coverage.rdd.count == 3)
+    assert(coverage.dataset.count == 3)
+
+    // go to dataset and save as parquet
+    val outputFile2 = tmpLocation(".adam")
+    coverageRDD.transformDataset(ds => ds).save(outputFile2, false, false)
+    val coverage2 = sc.loadCoverage(outputFile2)
+    assert(coverage2.rdd.count == 3)
+    assert(coverage2.dataset.count == 3)
+
+    // load as features, force to dataset, convert to coverage, and count
+    val features2Ds = sc.loadFeatures(outputFile2)
+      .transformDataset(ds => ds) // no-op, force to dataset
```

Comment:
`outputFile2` is a Parquet file; on calling `sc.loadFeatures`, we get a `ParquetUnboundFeatureRDD`. By calling `transformDataset`, we force the `FeatureRDD` to be bound to a `dataset`.